CMakeLists.txt

```
@@ -306,8 +306,17 @@ if (BUILD_TESTING)
         )
     endif()
 
-    set(SOUNDFONT "$ENV{TEMP}/soundfont.dls")
+    if(DEFINED ENV{TEMP})
+        set(TEMPDIR "$ENV{TEMP}")
+    elseif(DEFINED ENV{XDG_RUNTIME_DIR})
+        set(TEMPDIR "$ENV{XDG_RUNTIME_DIR}")
+    else()
+        error("Cannot find a temporary directory. Define the TEMP environment variable.")
```

```suggestion
        message(FATAL_ERROR "Cannot find a temporary directory. Define the TEMP environment variable.")
```
There is no `errror()` standard cmake macro.